### ISSUE TITLE
Feature Request: Show CPU Core Count in Summary

### ISSUE THREAD

**Issue by maieredv-manuel** [amidaware/tacticalrmm#1715]
**Is your feature request related to a problem? Please describe.**
In the Summary the CPU Type is shown, but saadly not the Core Count - sure, on a physical Machine you can search after the CPU Type, but on a VM there are virtual CPU Cores, so you also can have a i7-9700K with only 4 Cores on a VM.

**Describe the solution you'd like**
Show the CPU Core Count on the Summary

**Describe alternatives you've considered**
Actually ive run two Scripts:
one for the Cores: _wmic cpu get NumberOfCores_
and one for the Logical Cores (Threads): _wmic cpu get NumberOfLogicalProcessors_
Both scripts are running automaticaly once a day and pushes the answers to Custom Fields.
Its OK, but it would be nicer if these infos are in one Line with the CPU, RAM and so on.

**Comment by wh1te909** [comment]
Like this?

![jGn4zsUXUU](https://github.com/amidaware/tacticalrmm/assets/7434746/1d4ae10a-cd30-4c1f-880a-0cc8e067adbb)

**Comment by wh1te909** [comment]
will be in next release

**Comment by maieredv-manuel** [comment]
Thank you! 🙂

**Comment by wh1te909** [comment]
yes will try to get it in the next release

**Comment by P6g9YHK6** [comment]
Microsoft licencing works off number of sockets on devices can we please also have another /xS for the numbers of processor on the device ?

**Comment by dinger1986** [comment]
> Microsoft licencing works off number of sockets on devices can we please also have another /xS for the numbers of processor on the device ?

its already there, if its more than 1 it shows as 2 x or whatever 

![image](https://github.com/amidaware/tacticalrmm/assets/7244447/6c32f0dd-adf6-4031-ab46-b651acb07347)

**Comment by P6g9YHK6** [comment]
> > Microsoft licencing works off number of sockets on devices can we please also have another /xS for the numbers of processor on the device ?
> 
> its already there, if its more than 1 it shows as 2 x or whatever
> 
> ![image](https://private-user-images.githubusercontent.com/7244447/311541465-6c32f0dd-adf6-4031-ab46-b651acb07347.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTAxMDIwMDUsIm5iZiI6MTcxMDEwMTcwNSwicGF0aCI6Ii83MjQ0NDQ3LzMxMTU0MTQ2NS02YzMyZjBkZC1hZGY2LTQwMzEtYWI0Ni1iNjUxYWNiMDczNDcucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MDMxMCUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDAzMTBUMjAxNTA1WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9N2MyNzQ2ZjI3MjRiNmUwYzU1YTNkMjBkMzA0OTZkZGFiMWRiNmM5ZTBiZTJkNjljZTM0YmVmZDU0OTMyMDQ1YyZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.n0V6if3-4Nf4qDiAMOmQvBm2UDOmiwKHS-PbmOvrLQY)

yes you are right i had the missconception that it was the core count.

**Comment by vithusel** [comment]
@silversword411 this should stay open. wh1te909 mentioned he would get the Linux portion of this completed for next release in response to my previous query higher up ☺️
